cfme/configure/access_control/__init__.py

```diff
@@ -95,10 +95,7 @@ class DetailsUserView(ConfigurationView):
     def is_displayed(self):
         return (
             self.title.text == 'EVM User "{}"'.format(self.context['object'].name) and
-            self.accordions.accesscontrol.is_opened and
-            # tree.currently_selected returns a list of strings with each item being the text of
-            # each level of the accordion. Last element should be the User's name
-            self.accordions.accesscontrol.tree.currently_selected[-1] == self.context['object'].name
+            self.accordions.accesscontrol.is_opened
```

Comment:
You'll want to remove similar checks in DetailsRoleView and DetailsGroupView to fix the same type of failures

Reply:
Done, good catch, thanks @LandoCalrizzian !